web/src/pages/Setting/setting-account.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
```

Comment:
You should remove this, this directory is already pushed in current under a new name